### DISCUSSION TITLE
reverse proxy conf for this?

### DISCUSSION THREAD

**Discussion by danktankk** [sct/overseerr#299] [General]
Most of my containers are set up using nginx/reverse proxy with letsencrypt.  I am sure you will find a great number of folks from the UNRAID community doing the same.  Do you have a default "overseerr.conf" file for those of us that prefer to access this over https?  

Im certain this would be figured out eventually, but I am just not that good at it.  Thanks for any input.  

Getting the bugs out is more important than this, but I just wanted to make a note of it.  Great app btw!

**Comment by Drawmonster** [comment]
[overseerr.subdomain.conf.txt](https://github.com/sct/overseerr/files/5694006/overseerr.subdomain.conf.txt)

Confirmed working with LE/SWAG using a subdomain reverse proxy. Just remove `.txt` and drop in your `site-confs` folder.

**Reply by danktankk** [reply]
can confirm.  works like a charm.  thanks!  also using swag per lsio

**Reply by Swiftnesses** [reply]
Sadly I don't use a sub-domain, but a folder location, i.e. dashboard.xxx.com/overseerr/ - is it possible to specify a 'base url' like in Radarr and Sonarr?

**Reply by sct** [reply]
> Sadly I don't use a sub-domain, but a folder location, i.e. dashboard.xxx.com/overseerr/ - is it possible to specify a 'base url' like in Radarr and Sonarr?

@Swiftnesses Follow this issue for updates https://github.com/sct/overseerr/issues/274

**Comment by ghost** [comment]
You can see more [examples](https://docs.overseerr.dev/extending-overseerr/reverse-proxy-examples) of reverse proxy configurations by visiting the [Overseerr Documentation](https://docs.overseerr.dev).
